_posts/2017-01-21-Wonders-of-Static-Linking.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: "Wonders Of Static Linking"
+date: 2017-01-21 20:37:00
+author: lordPoseidon
+---
+
+Hi, this is Nimit Bhardwaj, a student of Computer Science, we all in the journey of computer science got the chance to learn the C Programming language, yes we all, not all although, learning C as out first programming language taught us many things, many things which are related to the working of the basics of the programming.
+
+We are taught how to compile a program, yes very easy process, most of us think its just pressing the **Compile** button or pressing the **Compile and Execute** button for those who use Dev-C/C++ or just use `gcc` who use linux. Its a good defination for a layman, but for a programmer, yes for us, its not enough. For the coders(some basic coders), like us, there are 4 steps of the compilation these are
```

Comment:
Spelling of definition is wrong here. `its` should be `it's`. I would remove basic coders and the brackets. Full stop after compilation.